docs/playroom/snippets.ts

```diff
@@ -1184,14 +1184,14 @@ const snippets: Array<Snippet> = [
 				<SummaryListItemTerm>First name</SummaryListItemTerm>
 				<SummaryListItemDescription>Will</SummaryListItemDescription>
 				<SummaryListItemAction>
-					<TextLink href="#">Change</TextLink>
+					<TextLink href="#">Change <VisuallyHidden>first name</VisuallyHidden></TextLink>
```

Comment:
I'm curious if there's a reason you didn't put the space within the `<VisuallyHidden>` component?

Reply:
No major reason, but I do feel it's easier to grok that there is a space to separate the content and since whitespace isn't rendered by default, it doesn't affect the standard layout